diff -urNp requests-2.5.3.org/requests/adapters.py requests-2.5.3/requests/adapters.py
--- requests-2.5.3.org/requests/adapters.py	2014-12-01 22:02:39.000000000 +0100
+++ requests-2.5.3/requests/adapters.py	2015-03-06 20:15:36.361707968 +0100
@@ -11,22 +11,22 @@ and maintain connections.
 import socket
 
 from .models import Response
-from .packages.urllib3 import Retry
-from .packages.urllib3.poolmanager import PoolManager, proxy_from_url
-from .packages.urllib3.response import HTTPResponse
-from .packages.urllib3.util import Timeout as TimeoutSauce
+from urllib3 import Retry
+from urllib3.poolmanager import PoolManager, proxy_from_url
+from urllib3.response import HTTPResponse
+from urllib3.util import Timeout as TimeoutSauce
 from .compat import urlparse, basestring
 from .utils import (DEFAULT_CA_BUNDLE_PATH, get_encoding_from_headers,
                     prepend_scheme_if_needed, get_auth_from_url, urldefragauth)
 from .structures import CaseInsensitiveDict
-from .packages.urllib3.exceptions import ConnectTimeoutError
-from .packages.urllib3.exceptions import HTTPError as _HTTPError
-from .packages.urllib3.exceptions import MaxRetryError
-from .packages.urllib3.exceptions import ProxyError as _ProxyError
-from .packages.urllib3.exceptions import ProtocolError
-from .packages.urllib3.exceptions import ReadTimeoutError
-from .packages.urllib3.exceptions import SSLError as _SSLError
-from .packages.urllib3.exceptions import ResponseError
+from urllib3.exceptions import ConnectTimeoutError
+from urllib3.exceptions import HTTPError as _HTTPError
+from urllib3.exceptions import MaxRetryError
+from urllib3.exceptions import ProxyError as _ProxyError
+from urllib3.exceptions import ProtocolError
+from urllib3.exceptions import ReadTimeoutError
+from urllib3.exceptions import SSLError as _SSLError
+from urllib3.exceptions import ResponseError
 from .cookies import extract_cookies_to_jar
 from .exceptions import (ConnectionError, ConnectTimeout, ReadTimeout, SSLError,
                          ProxyError, RetryError)
diff -urNp requests-2.5.3.org/requests/compat.py requests-2.5.3/requests/compat.py
--- requests-2.5.3.org/requests/compat.py	2015-02-24 17:27:00.000000000 +0100
+++ requests-2.5.3/requests/compat.py	2015-03-06 20:15:36.365041382 +0100
@@ -4,7 +4,7 @@
 pythoncompat
 """
 
-from .packages import chardet
+import charade as chardet
 
 import sys
 
@@ -39,7 +39,7 @@ if is_py2:
     import cookielib
     from Cookie import Morsel
     from StringIO import StringIO
-    from .packages.urllib3.packages.ordered_dict import OrderedDict
+    from urllib3.packages.ordered_dict import OrderedDict
 
     builtin_str = str
     bytes = str
diff -urNp requests-2.5.3.org/requests/exceptions.py requests-2.5.3/requests/exceptions.py
--- requests-2.5.3.org/requests/exceptions.py	2014-12-01 22:02:39.000000000 +0100
+++ requests-2.5.3/requests/exceptions.py	2015-03-06 20:15:36.368374796 +0100
@@ -7,7 +7,7 @@ requests.exceptions
 This module contains the set of Requests' exceptions.
 
 """
-from .packages.urllib3.exceptions import HTTPError as BaseHTTPError
+from urllib3.exceptions import HTTPError as BaseHTTPError
 
 
 class RequestException(IOError):
diff -urNp requests-2.5.3.org/requests/__init__.py requests-2.5.3/requests/__init__.py
--- requests-2.5.3.org/requests/__init__.py	2015-02-24 17:33:01.000000000 +0100
+++ requests-2.5.3/requests/__init__.py	2015-03-06 20:15:36.358374555 +0100
@@ -50,7 +50,7 @@ __copyright__ = 'Copyright 2015 Kenneth
 
 # Attempt to enable urllib3's SNI support, if possible
 try:
-    from .packages.urllib3.contrib import pyopenssl
+    from urllib3.contrib import pyopenssl
     pyopenssl.inject_into_urllib3()
 except ImportError:
     pass
diff -urNp requests-2.5.3.org/requests/models.py requests-2.5.3/requests/models.py
--- requests-2.5.3.org/requests/models.py	2014-12-23 18:43:04.000000000 +0100
+++ requests-2.5.3/requests/models.py	2015-03-06 20:15:36.375041624 +0100
@@ -16,10 +16,10 @@ from .structures import CaseInsensitiveD
 
 from .auth import HTTPBasicAuth
 from .cookies import cookiejar_from_dict, get_cookie_header
-from .packages.urllib3.fields import RequestField
-from .packages.urllib3.filepost import encode_multipart_formdata
-from .packages.urllib3.util import parse_url
-from .packages.urllib3.exceptions import (
+from urllib3.fields import RequestField
+from urllib3.filepost import encode_multipart_formdata
+from urllib3.util import parse_url
+from urllib3.exceptions import (
     DecodeError, ReadTimeoutError, ProtocolError, LocationParseError)
 from .exceptions import (
     HTTPError, MissingSchema, InvalidURL, ChunkedEncodingError,
diff -urNp requests-2.5.3.org/requests/packages/__init__.py requests-2.5.3/requests/packages/__init__.py
--- requests-2.5.3.org/requests/packages/__init__.py	2015-02-24 17:27:00.000000000 +0100
+++ requests-2.5.3/requests/packages/__init__.py	2015-03-06 20:15:36.375041624 +0100
@@ -65,7 +65,7 @@ class VendorAlias(object):
             finally:
                 # Re-add any additions to sys.meta_path that were made while
                 # during the import we just did, otherwise things like
-                # requests.packages.urllib3.poolmanager will fail.
+                # urllib3.poolmanager will fail.
                 for m in sys.meta_path:
                     if m not in real_meta_path:
                         real_meta_path.append(m)
diff -urNp requests-2.5.3.org/requests/sessions.py requests-2.5.3/requests/sessions.py
--- requests-2.5.3.org/requests/sessions.py	2014-12-01 22:02:39.000000000 +0100
+++ requests-2.5.3/requests/sessions.py	2015-03-06 20:15:36.488377701 +0100
@@ -21,7 +21,7 @@ from .hooks import default_hooks, dispat
 from .utils import to_key_val_list, default_headers, to_native_string
 from .exceptions import (
     TooManyRedirects, InvalidSchema, ChunkedEncodingError, ContentDecodingError)
-from .packages.urllib3._collections import RecentlyUsedContainer
+from urllib3._collections import RecentlyUsedContainer
 from .structures import CaseInsensitiveDict
 
 from .adapters import HTTPAdapter
diff -urNp requests-2.5.3.org/setup.py requests-2.5.3/setup.py
--- requests-2.5.3.org/setup.py	2014-09-10 17:30:41.000000000 +0200
+++ requests-2.5.3/setup.py	2015-03-06 20:16:15.829329894 +0100
@@ -18,13 +18,6 @@ if sys.argv[-1] == 'publish':
 
 packages = [
     'requests',
-    'requests.packages',
-    'requests.packages.chardet',
-    'requests.packages.urllib3',
-    'requests.packages.urllib3.packages',
-    'requests.packages.urllib3.contrib',
-    'requests.packages.urllib3.util',
-    'requests.packages.urllib3.packages.ssl_match_hostname',
 ]
 
 requires = []
diff -urNp requests-2.5.3.org/test_requests.py requests-2.5.3/test_requests.py
--- requests-2.5.3.org/test_requests.py	2015-02-24 17:27:00.000000000 +0100
+++ requests-2.5.3/test_requests.py	2015-03-06 20:15:36.498377943 +0100
@@ -1589,7 +1589,7 @@ def test_prepare_unicode_url():
 
 
 def test_urllib3_retries():
-    from requests.packages.urllib3.util import Retry
+    from urllib3.util import Retry
     s = requests.Session()
     s.mount('http://', HTTPAdapter(max_retries=Retry(
         total=2, status_forcelist=[500]
