diff -urN requests-2.4.3.org/requests/adapters.py requests-2.4.3/requests/adapters.py
--- requests-2.4.3.org/requests/adapters.py	2014-10-06 11:40:10.000000000 +0200
+++ requests-2.4.3/requests/adapters.py	2014-10-21 17:40:54.823464764 +0200
@@ -11,21 +11,21 @@
 import socket
 
 from .models import Response
-from .packages.urllib3 import Retry
-from .packages.urllib3.poolmanager import PoolManager, proxy_from_url
-from .packages.urllib3.response import HTTPResponse
-from .packages.urllib3.util import Timeout as TimeoutSauce
+from urllib3 import Retry
+from urllib3.poolmanager import PoolManager, proxy_from_url
+from urllib3.response import HTTPResponse
+from urllib3.util import Timeout as TimeoutSauce
 from .compat import urlparse, basestring
 from .utils import (DEFAULT_CA_BUNDLE_PATH, get_encoding_from_headers,
                     prepend_scheme_if_needed, get_auth_from_url, urldefragauth)
 from .structures import CaseInsensitiveDict
-from .packages.urllib3.exceptions import ConnectTimeoutError
-from .packages.urllib3.exceptions import HTTPError as _HTTPError
-from .packages.urllib3.exceptions import MaxRetryError
-from .packages.urllib3.exceptions import ProxyError as _ProxyError
-from .packages.urllib3.exceptions import ProtocolError
-from .packages.urllib3.exceptions import ReadTimeoutError
-from .packages.urllib3.exceptions import SSLError as _SSLError
+from urllib3.exceptions import ConnectTimeoutError
+from urllib3.exceptions import HTTPError as _HTTPError
+from urllib3.exceptions import MaxRetryError
+from urllib3.exceptions import ProxyError as _ProxyError
+from urllib3.exceptions import ProtocolError
+from urllib3.exceptions import ReadTimeoutError
+from urllib3.exceptions import SSLError as _SSLError
 from .cookies import extract_cookies_to_jar
 from .exceptions import (ConnectionError, ConnectTimeout, ReadTimeout, SSLError,
                          ProxyError)
diff -urN requests-2.4.3.org/requests/compat.py requests-2.4.3/requests/compat.py
--- requests-2.4.3.org/requests/compat.py	2014-08-19 18:52:58.000000000 +0200
+++ requests-2.4.3/requests/compat.py	2014-10-21 17:40:54.830131661 +0200
@@ -4,7 +4,7 @@
 pythoncompat
 """
 
-from .packages import chardet
+import charade as chardet
 
 import sys
 
@@ -91,7 +91,7 @@
     import cookielib
     from Cookie import Morsel
     from StringIO import StringIO
-    from .packages.urllib3.packages.ordered_dict import OrderedDict
+    from urllib3.packages.ordered_dict import OrderedDict
 
     builtin_str = str
     bytes = str
diff -urN requests-2.4.3.org/requests/exceptions.py requests-2.4.3/requests/exceptions.py
--- requests-2.4.3.org/requests/exceptions.py	2014-10-05 18:53:35.000000000 +0200
+++ requests-2.4.3/requests/exceptions.py	2014-10-21 17:40:54.836798554 +0200
@@ -7,7 +7,7 @@
 This module contains the set of Requests' exceptions.
 
 """
-from .packages.urllib3.exceptions import HTTPError as BaseHTTPError
+from urllib3.exceptions import HTTPError as BaseHTTPError
 
 
 class RequestException(IOError):
diff -urN requests-2.4.3.org/requests/__init__.py requests-2.4.3/requests/__init__.py
--- requests-2.4.3.org/requests/__init__.py	2014-10-06 11:44:17.000000000 +0200
+++ requests-2.4.3/requests/__init__.py	2014-10-21 17:40:54.820131316 +0200
@@ -50,7 +50,7 @@
 
 # Attempt to enable urllib3's SNI support, if possible
 try:
-    from .packages.urllib3.contrib import pyopenssl
+    from urllib3.contrib import pyopenssl
     pyopenssl.inject_into_urllib3()
 except ImportError:
     pass
diff -urN requests-2.4.3.org/requests/models.py requests-2.4.3/requests/models.py
--- requests-2.4.3.org/requests/models.py	2014-10-06 11:40:10.000000000 +0200
+++ requests-2.4.3/requests/models.py	2014-10-21 17:40:54.840132000 +0200
@@ -16,10 +16,10 @@
 
 from .auth import HTTPBasicAuth
 from .cookies import cookiejar_from_dict, get_cookie_header
-from .packages.urllib3.fields import RequestField
-from .packages.urllib3.filepost import encode_multipart_formdata
-from .packages.urllib3.util import parse_url
-from .packages.urllib3.exceptions import (
+from urllib3.fields import RequestField
+from urllib3.filepost import encode_multipart_formdata
+from urllib3.util import parse_url
+from urllib3.exceptions import (
     DecodeError, ReadTimeoutError, ProtocolError)
 from .exceptions import (
     HTTPError, RequestException, MissingSchema, InvalidURL, 
diff -urN requests-2.4.3.org/setup.py requests-2.4.3/setup.py
--- requests-2.4.3.org/setup.py	2014-09-09 18:31:17.000000000 +0200
+++ requests-2.4.3/setup.py	2014-10-21 17:41:51.558737269 +0200
@@ -18,13 +18,6 @@
 
 packages = [
     'requests',
-    'requests.packages',
-    'requests.packages.chardet',
-    'requests.packages.urllib3',
-    'requests.packages.urllib3.packages',
-    'requests.packages.urllib3.contrib',
-    'requests.packages.urllib3.util',
-    'requests.packages.urllib3.packages.ssl_match_hostname',
 ]
 
 requires = []
