diff -urN requests-2.5.0.org/requests/adapters.py requests-2.5.0/requests/adapters.py
--- requests-2.5.0.org/requests/adapters.py	2014-12-01 22:02:39.000000000 +0100
+++ requests-2.5.0/requests/adapters.py	2014-12-04 11:09:50.937837828 +0100
@@ -11,22 +11,22 @@
 import socket
 
 from .models import Response
-from .packages.urllib3 import Retry
-from .packages.urllib3.poolmanager import PoolManager, proxy_from_url
-from .packages.urllib3.response import HTTPResponse
-from .packages.urllib3.util import Timeout as TimeoutSauce
+from urllib3 import Retry
+from urllib3.poolmanager import PoolManager, proxy_from_url
+from urllib3.response import HTTPResponse
+from urllib3.util import Timeout as TimeoutSauce
 from .compat import urlparse, basestring
 from .utils import (DEFAULT_CA_BUNDLE_PATH, get_encoding_from_headers,
                     prepend_scheme_if_needed, get_auth_from_url, urldefragauth)
 from .structures import CaseInsensitiveDict
-from .packages.urllib3.exceptions import ConnectTimeoutError
-from .packages.urllib3.exceptions import HTTPError as _HTTPError
-from .packages.urllib3.exceptions import MaxRetryError
-from .packages.urllib3.exceptions import ProxyError as _ProxyError
-from .packages.urllib3.exceptions import ProtocolError
-from .packages.urllib3.exceptions import ReadTimeoutError
-from .packages.urllib3.exceptions import SSLError as _SSLError
-from .packages.urllib3.exceptions import ResponseError
+from urllib3.exceptions import ConnectTimeoutError
+from urllib3.exceptions import HTTPError as _HTTPError
+from urllib3.exceptions import MaxRetryError
+from urllib3.exceptions import ProxyError as _ProxyError
+from urllib3.exceptions import ProtocolError
+from urllib3.exceptions import ReadTimeoutError
+from urllib3.exceptions import SSLError as _SSLError
+from urllib3.exceptions import ResponseError
 from .cookies import extract_cookies_to_jar
 from .exceptions import (ConnectionError, ConnectTimeout, ReadTimeout, SSLError,
                          ProxyError, RetryError)
diff -urN requests-2.5.0.org/requests/compat.py requests-2.5.0/requests/compat.py
--- requests-2.5.0.org/requests/compat.py	2014-08-29 21:37:47.000000000 +0200
+++ requests-2.5.0/requests/compat.py	2014-12-04 11:09:50.944504630 +0100
@@ -4,7 +4,7 @@
 pythoncompat
 """
 
-from .packages import chardet
+import charade as chardet
 
 import sys
 
@@ -91,7 +91,7 @@
     import cookielib
     from Cookie import Morsel
     from StringIO import StringIO
-    from .packages.urllib3.packages.ordered_dict import OrderedDict
+    from urllib3.packages.ordered_dict import OrderedDict
 
     builtin_str = str
     bytes = str
diff -urN requests-2.5.0.org/requests/exceptions.py requests-2.5.0/requests/exceptions.py
--- requests-2.5.0.org/requests/exceptions.py	2014-12-01 22:02:39.000000000 +0100
+++ requests-2.5.0/requests/exceptions.py	2014-12-04 11:09:50.947838039 +0100
@@ -7,7 +7,7 @@
 This module contains the set of Requests' exceptions.
 
 """
-from .packages.urllib3.exceptions import HTTPError as BaseHTTPError
+from urllib3.exceptions import HTTPError as BaseHTTPError
 
 
 class RequestException(IOError):
diff -urN requests-2.5.0.org/requests/__init__.py requests-2.5.0/requests/__init__.py
--- requests-2.5.0.org/requests/__init__.py	2014-12-02 00:22:46.000000000 +0100
+++ requests-2.5.0/requests/__init__.py	2014-12-04 11:09:50.934504432 +0100
@@ -50,7 +50,7 @@
 
 # Attempt to enable urllib3's SNI support, if possible
 try:
-    from .packages.urllib3.contrib import pyopenssl
+    from urllib3.contrib import pyopenssl
     pyopenssl.inject_into_urllib3()
 except ImportError:
     pass
diff -urN requests-2.5.0.org/requests/models.py requests-2.5.0/requests/models.py
--- requests-2.5.0.org/requests/models.py	2014-11-16 18:39:53.000000000 +0100
+++ requests-2.5.0/requests/models.py	2014-12-04 11:09:50.951171470 +0100
@@ -16,10 +16,10 @@
 
 from .auth import HTTPBasicAuth
 from .cookies import cookiejar_from_dict, get_cookie_header
-from .packages.urllib3.fields import RequestField
-from .packages.urllib3.filepost import encode_multipart_formdata
-from .packages.urllib3.util import parse_url
-from .packages.urllib3.exceptions import (
+from urllib3.fields import RequestField
+from urllib3.filepost import encode_multipart_formdata
+from urllib3.util import parse_url
+from urllib3.exceptions import (
     DecodeError, ReadTimeoutError, ProtocolError)
 from .exceptions import (
     HTTPError, RequestException, MissingSchema, InvalidURL, 
diff -urN requests-2.5.0.org/requests/sessions.py requests-2.5.0/requests/sessions.py
--- requests-2.5.0.org/requests/sessions.py	2014-12-01 22:02:39.000000000 +0100
+++ requests-2.5.0/requests/sessions.py	2014-12-04 11:09:51.081174694 +0100
@@ -21,7 +21,7 @@
 from .utils import to_key_val_list, default_headers, to_native_string
 from .exceptions import (
     TooManyRedirects, InvalidSchema, ChunkedEncodingError, ContentDecodingError)
-from .packages.urllib3._collections import RecentlyUsedContainer
+from urllib3._collections import RecentlyUsedContainer
 from .structures import CaseInsensitiveDict
 
 from .adapters import HTTPAdapter
diff -urN requests-2.5.0.org/setup.py requests-2.5.0/setup.py
--- requests-2.5.0.org/setup.py	2014-09-10 17:30:41.000000000 +0200
+++ requests-2.5.0/setup.py	2014-12-04 11:10:15.661783566 +0100
@@ -18,13 +18,6 @@
 
 packages = [
     'requests',
-    'requests.packages',
-    'requests.packages.chardet',
-    'requests.packages.urllib3',
-    'requests.packages.urllib3.packages',
-    'requests.packages.urllib3.contrib',
-    'requests.packages.urllib3.util',
-    'requests.packages.urllib3.packages.ssl_match_hostname',
 ]
 
 requires = []
diff -urN requests-2.5.0.org/test_requests.py requests-2.5.0/test_requests.py
--- requests-2.5.0.org/test_requests.py	2014-12-01 23:26:18.000000000 +0100
+++ requests-2.5.0/test_requests.py	2014-12-04 11:09:51.094508361 +0100
@@ -1532,7 +1532,7 @@
 
 
 def test_urllib3_retries():
-    from requests.packages.urllib3.util import Retry
+    from urllib3.util import Retry
     s = requests.Session()
     s.mount('https://', HTTPAdapter(max_retries=Retry(
         total=2, status_forcelist=[500]
