--- requests-2.4.1/requests/certs.py~	2014-08-19 18:52:58.000000000 +0200
+++ requests-2.4.1/requests/certs.py	2014-09-24 15:15:55.773227329 +0200
@@ -18,8 +18,8 @@
 except ImportError:
     def where():
         """Return the preferred certificate bundle."""
-        # vendored bundle inside Requests
-        return os.path.join(os.path.dirname(__file__), 'cacert.pem')
+        # system certificates from ca-certificates package
+        return "/etc/certs/ca-certificates.crt"
 
 if __name__ == '__main__':
     print(where())
